sc/cdp_expression_macosko.tab.gz.info

```diff
@@ -17,9 +17,9 @@
         "drop-seq"
     ],
     "target": null,
-    "title": "Mouse retinal bipolar neurons (unlabelled)",
-    "url": "http://butler.fri.uni-lj.si/datasets/sc/cdp_expression_macosko.tab.gz",
-    "variables": 6861,
+    "title": "Mouse retinal bipolar neurons (DropSeq)",
+    "url": "http://bubble.fri.uni-lj.si/zp/singlecell/datasets/cdp_expression_macosko.tab.gz",
```

Comment:
If you are changing the file, version should also be incremented.
E.g. 1.1 for a minor change or 2.0 if the data changes significantly.